docs/developers/mobile.rst

```diff
@@ -39,26 +41,11 @@ tested with at least current versions of iOS Safari and Chrome for Android.
 
       make dev
 
-#. Set the :envvar:`SIDEBAR_APP_URL` and :envvar:`PACKAGE_SERVER_HOSTNAME`
-   environment variables to load assets from this hostname and start the dev
-   server:
-
-   .. code-block:: sh
-
-      # In the client repository
-
-      # Set URL which sidebar app ("app.html") is loaded from
-      export SIDEBAR_APP_URL=http://<HOSTNAME>:5000/app.html
-      # Set hostname used when generating client asset URLs
-      export PACKAGE_SERVER_HOSTNAME=<HOSTNAME>
-
-      make dev
-
-   .. tip::
+#. Make sure the "Redirect URL" of the OAuth client associated with your
+   development client matches `<HOSTNAME>`. You can configure the OAuth clients
+   registered with h at http://localhost:5000/admin/oauthclients.
```

Comment:
Pardon my `rst` naivete...should the URL on this line be formatted in some way?

Reply:
It doesn't seem necessary. It gets auto-linkified when viewed using `make docs`.